_partials/_livesync-limitations.md

```diff
@@ -14,3 +14,29 @@
   target $SERVICE_LONG before syncing the table.
 
 * There is WAL volume growth on the source $PG instance during large table copy.
+
+* Continuous Aggregates Invalidation
+
+  LiveSync uses `session_replication_role=replica` during data replication,
```

Comment:
To prevent table triggers from firing, LiveSync uses `session_replication_role=replica` during data replication. This includes the internal triggers that mark continuous aggregates as invalid when the underlying data changes.